README.md

```diff
@@ -256,6 +273,8 @@ Be also to then run:
 
 This will fetch updates to the DNS zone.
 
+`inventory_hostname` MUST NOT be renamed.
+
```

Comment:
@hellais @bassosimone do you confirm, that we've agreed on that naming policy three and a half months ago? :)

Reply:
I am not sure. What do you mean?